.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -0,0 +1,177 @@
+"""Code Coverage Disable Checker Script.
```

Ruff (0.12.2)

The file is executable but no shebang is present

(EXE002)